hw/ip/otbn/doc/_index.md

```diff
@@ -372,14 +372,38 @@ The number of entries in the loop stack is implementation-dependent.
 
 ## Design Details
 
-<div class="bd-callout bd-callout-warning">
-  <h5>Note</h5>
+### Error Handling and Reporting
 
-  To be filled in as we create the implementation.
-</div>
+By design, OTBN is a simple processor and provides no error handling support to code that runs on it.
+Any errors will cause OTBN to stop executing and report the failure to the host processor.
```

Comment:
Remove this sentence?

When OTBN isn't running, it won't stop executing and also will not (necessarily) report anything to the host processor.

Reply:
Yep, I agree. This sentence probably came along for the ride when I re-hashed things and I didn't notice. Thanks!